spotinst_sdk2/clients/ocean/__init__.py

```diff
@@ -460,8 +462,641 @@ def launch_nodes_in_vng(self, ocean_launch_spec_id: str, amount: int):
+
+        # Arguments
+        ocean_id (String): ID of the Ocean VNG
+
+        # Returns
+        (Object): Ocean Attach Load Balancers API response
+        """
+
+        load_balancers = aws_ocean.LoadBalancersRequest(load_balancers)
+
+        excluded_missing_dict = self.exclude_missing(json.loads(load_balancers.toJSON()))
+
+        formatted_missing_dict = self.convert_json(
+            excluded_missing_dict, self.underscore_to_camel)
+
+        body_json = json.dumps(formatted_missing_dict)
+
+        return self.send_put(
+            url=self.__base_ocean_cluster_url + "/" + ocean_id + "/loadBalancer/attach",
+            entity_name="ocean_aws_vng",
+            body=body_json
+        )
+
+    def detach_load_balancers(self, ocean_id, load_balancers):
+        """
+        Delete an Ocean Cluster
+
+        # Arguments
+        ocean_id (String): ID of the Ocean VNG
+
+        # Returns
+        (Object): Ocean Detach Load Balancers API response
+        """
+
+        load_balancers = aws_ocean.LoadBalancersRequest(load_balancers)
+
+        excluded_missing_dict = self.exclude_missing(json.loads(load_balancers.toJSON()))
+
+        formatted_missing_dict = self.convert_json(
+            excluded_missing_dict, self.underscore_to_camel)
+
+        body_json = json.dumps(formatted_missing_dict)
+
+        return self.send_put(
+            url=self.__base_ocean_cluster_url + "/" + ocean_id + "/loadBalancer/detach",
+            entity_name="ocean_aws_vng",
+            body=body_json
+        )    
+
+    def update_elastigroup_to_ocean(self, group_id: str):
+        """
+        Upgrade an Elastigroup with Kubernetes integration to Ocean for Kubernetes cluster. 
+        
+        # Arguments
+        group_id (str): Elastigroup identifier
+
+        # Returns
+        (Object): Ocean API response 
+        """
+
+        query_params = dict(groupId=group_id)
+
+        response = self.send_post_with_params(
+            body=None,
+            url=self.__base_ocean_cluster_url+"/import",
+            entity_name='ocean_aws',
+            user_query_params=query_params)
+
+        formatted_response = self.convert_json(response, 
+                                               self.camel_to_underscore)
+
+        return formatted_response["response"]["items"][0]
+
+    def detach_instances(self, ocean_id: str, detach_instances: aws_ocean.DetachInstances):
+        """
+        Detach instances from your Ocean cluster.
+
+        # Arguments
+        ocean_id (String): ID of the Ocean Cluster
+        detach_instances (DetachInstances): Detach Instances Request
+
+        # Returns
+        (Object): Detach Instances API response
+        """
+        detach_instances_request = aws_ocean.DetachInstancesRequest(detach_instances)
+
+        excluded_missing_dict = self.exclude_missing(json.loads(detach_instances_request.toJSON()))
+
+        formatted_missing_dict = self.convert_json_with_list_of_lists(
```

Comment:
convert_json_with_list_of_lists - This doesn't seem correct.

Reply:
changed to convert_json